bundles/aero.minova.rcp.constant/src/aero/minova/rcp/util/TimeUtil.java

```diff
@@ -69,6 +78,26 @@ public static Instant getTime(Instant now, String input, String timeUtilPref) {
 	}
 
 	public static Instant getTime(Instant now, String input, String timeUtilPref, Locale locale) {
+		try {
+			DateTimeFormatter dtf = DateTimeFormatter.ofPattern(timeUtilPref, locale);
+			LocalTime lt = LocalTime.parse(input.toLowerCase(), dtf);
+			LocalDateTime ldt = LocalDateTime.of(LocalDate.of(1900, 1, 1), lt);
+			now = ldt.toInstant(ZoneOffset.UTC);
+		} catch (Exception e) {
+			now = getAlternativeTime(now, input, timeUtilPref, locale);
+		}
+
+		return now;
+	}
+
+	public static Instant getAlternativeTime(Instant now, String input, String timeUtilPref, Locale locale) {
+		if (input.contains("am")) {
+			input = input.substring(0, input.indexOf("a")) + input.substring(input.indexOf("a") + 2);
+		} else if (input.contains("pm")) {
+			input = input.substring(0, input.indexOf("p")) + input.substring(input.indexOf("p") + 2);
+		}
+		System.out.println(input);
```

Comment:
Print entfernen

AM/PM sollte auch entfernt werden, wenn es groß geschrieben ist oder? 
(evtl den Input String einfach .toLowerCase() bevor du damit weiterarbeitest)

Reply:
Außerdem würde das auf deutsch nicht funktionieren oder? Bekommt man irgendwie die Strings für vormittag/nachmittag je nach Locale raus und kann sie stattdessen entfernen?

Reply:
Tatsächlich habe ich es versucht, aber keine Möglichkeit gefunden. Die Nullpointer tritt aber tatsächlich nur bei am/pm auf. Habe es auf Spanisch, Deutsch, Französisch getestet und nirgendwo sonst taucht der Fehler auf.

Reply:
Werde es in einem Kommentar dazu schreiben